#float[] duration 
int32 command #the number of the command
---
bool success      # Did it achieve it?